ocaml-lsp-server/src/custom_requests/custom_request.ml

```diff
@@ -0,0 +1,31 @@
+open Import
+
+type 't req_params_spec =
```

Comment:
mli?

Reply:
Added an mli file. I'm not 100% sure about how `req_params_spec` thing is going to work out, eg whether other contributors will understand how and why to use it, but it seems somewhat useful.